server/schedule/operator/operator_test.go

```diff
@@ -150,6 +150,14 @@ func (s *testOperatorSuite) TestOperator(c *C) {
 	c.Assert(op.CheckTimeout(), IsFalse)
 	SetOperatorStatusReachTime(op, STARTED, op.GetStartTime().Add(-FastOperatorWaitTime-time.Second))
 	c.Assert(op.CheckTimeout(), IsTrue)
+
+	// case2: check successful operator will return false not panic.
```

Comment:
If an operator is successful, it will also return directly in the previous implementation?